snare/server.py

```diff
@@ -89,8 +77,9 @@ def __init__(self, meta, run_args, snare_uuid, debug=False, keep_alive=75, **kwa
             app, loader=jinja2.FileSystemLoader(self.dir)
         )
         middleware = SnareMiddleware(
-            self.meta['/status_404']['hash'],
-            self.run_args.server_header
+            headers=self.meta['/status_404']['headers'],
```

Comment:
I got KeyError with old-cloned page, works perfectly after re-cloning, maybe it would be good to handle this problem? At least show some meaningful error and mention that re-cloning is needed

Reply:
I made the rest of the new code backward-compatible but I missed this part. I will change it so that if the headers are not available they will be ignored.